src/library-authoring/data/api.ts

```diff
@@ -384,6 +394,16 @@ export async function updateLibraryTeamMember(memberData: UpdateLibraryTeamMembe
   return camelCaseObject(data);
 }
 
+/**
+ * Get the block types metadata.
```

Comment:
```suggestion
 * Get the list of XBlock types that can be added to this library
```
The backend [calls this API](https://github.com/openedx/edx-platform/blob/9416fba6fa53da368a1efa5e7303eb49e8e434b9/openedx/core/djangoapps/content_libraries/urls.py#L38-L39) "Get the list of XBlock types that can be added to this library", and I think that's more clear. Maybe we should even rename this function to just `getBlockTypes` for simplicity. It's fine as is though.

Reply:
Updated https://github.com/openedx/frontend-app-authoring/pull/1653/commits/3263571b6cb8069ad7ee062d1d2593fa93245574